### ISSUE TITLE
Remove all the unused dependencies which are nowhere used

### ISSUE THREAD

**Issue by covalentbond** [codeforcauseorg/codeforcause.org#152]
There are many dependencies like "react-quill", "formik", "immutable", "react-draft-wysiwyg" and many more...

![image](https://user-images.githubusercontent.com/53327173/98581709-e3d35600-22e7-11eb-9047-65b55363c2ab.png)

**Comment by Abhishek-kumar09** [comment]
Thanks @covalentbond for raising this detailed issue.
> react-quill", "formik", "immutable", "react-draft-wysiwyg"

Ya, they are used no where.

For the image - These all are unused, but the similar ones are used within the code, Please assure that the code functionality is not compromised by removing them.

**Comment by covalentbond** [comment]
Those which are shown in the image have not been imported anywhere in the code. The image is a result of a library "depcheck" which shows the unused dependencies.

But it cannot be completely correct, as it showed "keycloak-js" as one of the unused dependencies as it's not imported. But "keycloack-js" is also not imported but actually used in one of the files.

The only way to this is checking manually each of the dependencies, whether it has been used anywhere in the project or not.
There might be even more which are not stated above

**Comment by Itsaadarsh** [comment]
Hello, there @Abhishek-kumar09 @covalentbond , is this issue still open? Because I'd love to work on this issue and resolve it.

**Comment by Abhishek-kumar09** [comment]
@Itsaadarsh  It's open.
But would you help in removing some console errors from the website first. This way you could learn more about codebase, and they are pretty easy too. You could remove few of them and make a PR. 
https://github.com/codeforcauseorg/codeforcause.org/issues/92

**Comment by Abhishek-kumar09** [comment]
Always feel free to tag me, as sometimes conversations are missed, and the fault is totally mine.

**Comment by Itsaadarsh** [comment]
Yes sure thing @Abhishek-kumar09 , but first I'll send (#152) PR then I'll move towards the next. Because I'm done removing all the unused dependencies, I'll clean up a little and send the PR asap.

**Comment by Abhishek-kumar09** [comment]
Great! thanks.
